addons/account/models/account_move_line.py

```diff
@@ -1335,6 +1335,27 @@ def check_field_access_rights(self, operation, field_names):
             result = [fname for fname in result if fname not in weirdos]
         return result
 
+    @api.model
+    def _get_default_read_fields(self):
```

Comment:
But so you are removing the extension of `check_field_access_rights`, right?

Reply:
The function is deprecated, so it can be removed (I was counting on doing that in the future and remove all `check_field_access_rights` at once).